extensions/amp-ad/0.1/test/test-amp-ad-3p-impl.js

```diff
@@ -210,14 +210,52 @@ function tests(name) {
           resizable: '',
         }, 'https://schema.org').then(element => {
           const impl = element.implementation_;
-          const attemptChangeSizeSpy = sandbox.spy(impl, 'attemptChangeSize');
-          impl.apiHandler_.updateSize_(217, 114);
-          expect(attemptChangeSizeSpy.callCount).to.equal(1);
-          expect(attemptChangeSizeSpy.firstCall.args[0]).to.equal(217);
-          expect(attemptChangeSizeSpy.firstCall.args[1]).to.equal(114);
+          const attemptChangeSize = sandbox.stub(impl, 'attemptChangeSize',
```

Comment:
I suggest we do the right thing in this PR. You don't need to move legacy tests over to the new file, but it's a good time to create such a separate unit test file, and have the new test in it. Otherwise, it's a waste of time to review and merge this PR in.

Meanwhile, it's better we test the external facing API of the `amp-ad-api-handler.js` but not internal private method. In this case, we should `window.postMessage('embed-size')`, instead of calling `updateSize_()`